cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/io/cdap/cdap/etl/common/DefaultMacroEvaluator.java

```diff
@@ -46,6 +46,7 @@ public DefaultMacroEvaluator(BasicArguments arguments, long logicalStartTime,
       LogicalStartTimeMacroEvaluator.FUNCTION_NAME, new LogicalStartTimeMacroEvaluator(logicalStartTime),
       SecureStoreMacroEvaluator.FUNCTION_NAME, new SecureStoreMacroEvaluator(namespace, secureStore),
       OAuthMacroEvaluator.FUNCTION_NAME, new OAuthMacroEvaluator(serviceDiscoverer),
+      OAuthAccessTokenMacroEvaluator.FUNCTION_NAME, new OAuthAccessTokenMacroEvaluator(serviceDiscoverer),
```

Comment:
We'll need to make sure we also add this macro evaluator everywhere the original one is used, including in the following places:
 - https://github.com/cdapio/cdap/blob/develop/cdap-app-templates/cdap-etl/cdap-data-pipeline-base/src/main/java/io/cdap/cdap/datapipeline/service/RemoteConnectionTaskBase.java#L86
 - https://github.com/cdapio/cdap/blob/develop/cdap-app-templates/cdap-etl/cdap-data-pipeline-base/src/main/java/io/cdap/cdap/datapipeline/service/ConnectionHandler.java#L517
 - https://github.com/cdapio/cdap/blob/develop/cdap-app-templates/cdap-etl/cdap-data-pipeline-base/src/main/java/io/cdap/cdap/datapipeline/service/RemoteValidationTask.java#L93
 - https://github.com/cdapio/cdap/blob/develop/cdap-app-templates/cdap-etl/cdap-data-pipeline-base/src/main/java/io/cdap/cdap/datapipeline/service/ValidationHandler.java#L164

These are used by validation and connection handlers.

Reply:
got'cha.  thanks for pointing this out.